pkg/build/docker/init.go

```diff
@@ -18,11 +18,14 @@ func Init() error {
 	}
 
 	// Enable execution of Docker images for other architectures
+	//nolint:gosec
 	cmd := exec.Command("docker", "run", "--privileged", "--rm",
-		"docker/binfmt:a7996909642ee92942dcd6cff44b9b95f08dad64")
+		"tonistiigi/binfmt", "--install", "all")
```

Comment:
Please pin that to a specifc version :)

Reply:
I tried to do that but unfortunately all the `buildkit` tags didn't work.
I'll try with `qemu`.